backend/app/usecase/changelog/changelog_test.go

```diff
@@ -448,6 +448,116 @@ func TestPersist_ViewChangeLog(t *testing.T) {
 	}
 }
 
+func TestPersist_GetChanges(t *testing.T) {
+	t.Parallel()
+
+	now := time.Now()
+	summaryMarkdown1 := "summary 1"
+	summaryMarkdown2 := "summary 2"
+	testCases := []struct {
+		name            string
+		changes         []entity.Change
+		expectedChanges []entity.Change
+		availableKeys   []keygen.Key
+		roles           map[string][]role.Role
+		user            entity.User
+		hasErr          bool
+	}{
+		{
+			name: "get full changelog successfully",
+			changes: []entity.Change{
+				{
+					ID:              "12345",
+					Title:           "title 1",
+					SummaryMarkdown: &summaryMarkdown1,
+				},
+				{
+					ID:              "54321",
+					Title:           "title 2",
+					SummaryMarkdown: &summaryMarkdown2,
+				},
+			},
+			expectedChanges: []entity.Change{
+				{
+					ID:              "12345",
+					Title:           "title 1",
+					SummaryMarkdown: &summaryMarkdown1,
+				},
+				{
+					ID:              "54321",
+					Title:           "title 2",
+					SummaryMarkdown: &summaryMarkdown2,
+				},
+			},
+			availableKeys: []keygen.Key{},
+			roles: map[string][]role.Role{
+				"alpha": {role.Admin},
```

Comment:
```suggestion
				"alpha": {role.ViewChange},
```
I think is better to test with the user that has the specific role